hapi-fees/src/main/java/com/hedera/services/usage/token/TokenBurnUsage.java

```diff
@@ -44,7 +44,7 @@ TokenBurnUsage self() {
 	public FeeData get() {
 		addEntityBpt();
 		addAmountBpt();
-		addTokenTransfersRecordRb(1, 1);
+		addTokenTransfersRecordRb(1, 1, 1);
```

Comment:
During TokenBurn we are NOT adding NFt transfer records in the transaction record (for now). It should be `1,1,0`

@joan41868 this should be updated when you implement the burn

Reply:
Corrected!